packages/components/icons/README.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: 'Icons'
```

Comment:
That'd be good, but what about the different GitHub and Storybook links etc.?

Reply:
Maybe we could merge storybook too, and link one readme from another?

Reply:
Maybe we could merge storybook too, and link one readme from another?

Reply:
Maybe we could merge storybook too, and link one readme from another?

Reply:
On second thought I think this might confuse users because the two package names are so similar, `icon` and `icons`. If everything is in the same README we might mike it harder to understand what to use when.